src/components/VolunteersForm.js

```diff
@@ -163,11 +163,11 @@ export default class VolunteersForm extends React.Component {
                         {/* // )} */}
 
                           <div className="form-group">
-                              <label id="Heard_about_us" htmlFor="herd">How did you hear about Software For Love? (Select all that Apply) {required_star}</label>
+                              <label id="Heard" htmlFor="heard">How did you hear about Software For Love? (Select all that Apply) {required_star}</label>
```

Comment:
Did the changes here work? If not, I would appreciate if you could push a commit reverting the changes as they were already on master

Reply:
Done!